pages/blog/hacktoberfest-summary-2020.md

```diff
@@ -0,0 +1,430 @@
+    byline: AsyncAPI Maintainer and Community Guardian
+---
+
+## tl;dr
+
+In October, we welcomed 26 new contributors with 70 pull requests (PRs) merged. It was an exhausting but also a fascinating experience.
+
+## Hacktoberfest Is Ok
+
+Don't be afraid of [Hacktoberfest](https://hacktoberfest.digitalocean.com/). It is an excellent event for both contributors and maintainers.
+
+There are [haters](https://blog.domenic.me/hacktoberfest/) that will tell you something different. My advice, follow [truncated mean](https://en.wikipedia.org/wiki/Truncated_mean) measure and always discard extreme opinions, especially if they call for boycotting:
+
+> Finally, and most importantly, we can remember that this is how DigitalOcean treats the open source maintainer community, and stay away from their products going forward
+
+Cancel culture at its best. The fact that someone is good at programming or works at Google or Facebook doesn't make them experts in everything. Remember that celebrities are not good candidates for a role model.
+
+There are no perfect events; there are no best solutions. There is always a place for improvement, but it should be followed by open, civilized discussion.
+
+Let me conclude by saying that I hope the "harm to the open source made by DigitalOcean" is not as significant as the harm that such hate does to open source by discouraging new open-source contributors. However, this is just speculation. How can I consider any of these things harmful if I did not conduct scientific research? I can only confirm that Hacktoberfest did not harm [AsyncAPI Initiative](https://www.asyncapi.com/). On the contrary, it was pretty neat.
+
+## Spam
+
+During the entire event, we had only two spam PRs. I can imagine that a much more popular and known project might have had more. Nevertheless, adding the **invalid** label and closing a PR is a super simple operation, three clicks.
+
+The definition of spam heavily depends on maintainers. For example, [this](https://github.com/facebook/react/pull/19953) is not invalid to me, because I don't think of grammar as "subjective nits".
+
+## Why We Participated in Hacktoberfest
+
+Our intentions were pretty clear from the very beginning. As I wrote in the [previous post](/blog/hacktoberfest-2020), we wanted to:
+ - Promote [AsyncAPI Initiative](https://www.asyncapi.com/) as a place where we work not only on the AsyncAPI specification, but also lots of tools
+ - Help members of the broader open-source community make their first contributions in a friendly environment
+
+My impression is that sometimes the open-source is perceived as a kind of elite gathering. This is quite often blocking people from joining because they feel they cannot help but rather waste others' time. I was there in the past, I thought the same. It's just another variation of damn [impostor syndrome](https://en.wikipedia.org/wiki/Impostor_syndrome). You can always help, no matter what your experience is.
```

Comment:
```suggestion
My impression is that sometimes the open-source is perceived as a kind of elite gathering. **This is quite often blocking people from joining because they feel they cannot help but rather waste others' time.** I was there in the past, I thought the same. It's just another variation of the damn [impostor syndrome](https://en.wikipedia.org/wiki/Impostor_syndrome). You can always help, no matter what your experience is.
```

Please, highlight this fucking sentence. I want everyone to read it 😄